package.json

```diff
@@ -13,7 +13,9 @@
     "url": "git://github.com/mcavage/node-asn1.git"
   },
   "main": "lib/index.js",
-  "dependencies": {},
+  "dependencies": {
+    "safer-buffer": "~2.0.2"
```

Comment:
This is not `^2.0.2` only because of npm bundled with Node.js 0.8 is too old and fails on that.
The code itself works on 0.8. I suggest changing to `~2.0.2` and dropping Node.js 0.8 from test matrix, but that could be done in a non-patch release, so I'm not including that to this PR.